src/OpenLoco/src/Map/MapGenerator/OriginalTerrainGenerator.cpp

```diff
@@ -91,8 +91,8 @@ namespace OpenLoco::World::MapGenerator
             {
                 const auto data = *src++;
                 heightMap[TilePos2(x, y)] = std::max(data, heightMap[TilePos2(x, y)]);
-                y--;
-                y %= heightMap.height - 1;
+                auto mod = heightMap.height - 1;
+                y = (--y + mod) % mod;
```

Comment:
[undefined in some versions of c++ (but not the one we use)](https://stackoverflow.com/questions/60937761/sequence-points-is-this-gcc-warning-a-bug)

Just use -1.
`y = (y - 1 + mod) % mod;`

Reply:
ah interesting, yep I can change it

Reply:
done